script/Arcueid/Day 13/13_00_ARC13_5.txt

```diff
@@ -172,6 +172,7 @@ Though I have to say―――that broken-down excuse of a princess is hardly wha
 [sha:4517f9cb0714b743912c248daab9af3d253a28f9]{
 -- Page 414, Offset 32849.
 -- 「一介の吸血鬼と変わらぬ真祖で手に入れても意味はない。
+// group pr check: getting my hands on vs obtain, the true ancestor not a
```

Comment:
Petrikow's comments were that he thought there was some nuance attached to "get my hands on" and that this line sounds like he's hunting for _any_ true ancestor when he specifically wants Arcueid.

I don't know if I agree that there's any change necessary here because:
a) I don't get any extra nuance from "get my hands on" whatsoever
b) i don't read this as him being particularly indifferent to which true ancestor he is, he's _describing_ arcueid as a True Ancestor who is no different from a vampire, he's not saying "well I'll certainly find an alternative".